src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/Constants/ApplicationInsightsDiagnosticConstants.cs

```diff
@@ -5,6 +5,6 @@ namespace Microsoft.Azure.WebJobs.Logging.ApplicationInsights
 {
     public static class ApplicationInsightsDiagnosticConstants
     {
-        public const string ApplicationInsightsDiagnosticSourcePrefix = "Microsoft.Azure.WebJobs.Logging.ApplicationInsights.";
+        public const string DiagnosticSourcePrefix = "Microsoft.Azure.WebJobs.Logging.ApplicationInsights.";
```

Comment:
Was this shipped before? If so, this is public and cannot be changed.

Reply:
reverted it.